examples/client/client.c

```diff
@@ -544,7 +544,8 @@ static int ClientBenchmarkConnections(WOLFSSL_CTX* ctx, char* host, word16 port,
     int version, int earlyData)
 {
     /* time passed in number of connects give average */
-    int times = benchmark, skip = times * 0.1;
+    int times = benchmark;
+    double skip = (double) times * 0.1;
```

Comment:
This rework for skip seems like a hack. Why not this?
`int skip = (int)((double)times * 0.1);`
And revert lines 579-582 below.